pkg/operator/controllers/gameserverbuild_controller.go

```diff
@@ -359,23 +384,34 @@ func (r *GameServerBuildReconciler) deleteNonActiveGameServers(ctx context.Conte
 	gsb *mpsv1alpha1.GameServerBuild,
 	gameServers *mpsv1alpha1.GameServerList,
 	totalNumberOfGameServersToDelete int) error {
-	deletedGameServersCount := 0
-	for i := 0; i < len(gameServers.Items) && deletedGameServersCount < totalNumberOfGameServersToDelete; i++ {
+	// an error channel for the go routines to write errors
+	errCh := make(chan error, totalNumberOfGameServersToDelete)
+	// a waitgroup for async deletion calls
+	var wg sync.WaitGroup
+	deletionCalls := 0
+	for i := 0; i < len(gameServers.Items) && deletionCalls < totalNumberOfGameServersToDelete; i++ {
 		gs := gameServers.Items[i]
-		// we're deleting only initializing/pending/standingBy servers, never touching active
```

Comment:
was this comment moved somewhere?

Reply:
I deleted it by accident, now it's back haha